### PR TITLE
Issue 4879

### PR DESCRIPTION
**Pull request by spalmstr**
Overview
----------------------------------------

This attempts to cure Issue 4879 - duplicate static declarations.

Before
----------------------------------------
The site crashes with:
```
PHP Fatal error:  Duplicate declaration of static variable $fields in <drupal root>/vendor/civicrm/civicrm-core/CRM/Mailing/Event/BAO/MailingEventForward.php on line 236
```
and
```
PHP Fatal error:  Duplicate declaration of static variable $fields in <drupal root>/vendor/civicrm/civicrm-core/CRM\Import/Parser.php on line 1131
```
After
----------------------------------------
The home page appears.

Technical Details
----------------------------------------
Removing the _static_ declaration on the offending lines in the offending files affects the cure.

Comments
----------------------------------------
_Anything else you would like the reviewer to note_

### CODE REVIEW

File: CRM/Import/Parser.php [hunk 1]
```diff
@@ -1128,7 +1128,7 @@ private function _civicrm_api3_deprecated_add_formatted_param(&$values, &$params
 
     // get the formatted location blocks into params - w/ 3.0 format, CRM-4605
     if (!empty($values['location_type_id'])) {
-      static $fields = NULL;
+      $fields = NULL;
       if ($fields == NULL) {
         $fields = [];
       }
```
Comment (colemanw) [inline CRM/Import/Parser.php:1134]: All 4 of these lines duplicate the start of the function and I think all 4 should be removed.
If you don't remove them all and only change L1131 then it changes the meaning.

File: CRM/Mailing/Event/BAO/MailingEventForward.php [hunk 2]
```diff
@@ -233,7 +233,7 @@ protected static function _civicrm_api3_deprecated_add_formatted_param(&$values,
 
     // get the formatted location blocks into params - w/ 3.0 format, CRM-4605
     if (!empty($values['location_type_id'])) {
-      static $fields = NULL;
+      $fields = NULL;
       if ($fields == NULL) {
         $fields = [];
       }
```
Comment (colemanw) [inline CRM/Mailing/Event/BAO/MailingEventForward.php:239]: Ditto for previous comment. In order to preserve the meaning I think all 4 lines should be removed.

Comment (spalmstr) [reply CRM/Mailing/Event/BAO/MailingEventForward.php]: Updated as per your suggestion and recommitted the file.

File: CRM/Import/Parser.php (hunk 1 above)
Comment (spalmstr) [reply CRM/Import/Parser.php]: Updated as per your suggestion and recommitted the file.

**Pull request closed without merging**